### DISCUSSION TITLE
Particle Life - 3D version (JavaScript)

### DISCUSSION THREAD

**Discussion by hunar4321** [hunar4321/particle-life#42] [Announcements]
Hi Everyone

The JavaScript version (2d) has improved by a lot thanks to @AnasQiblawi @skal65535 @dangarfield @rpominov @dlatikaynen . 
A live demo is here:
https://hunar4321.github.io/particle-life/particle_life.html#44225060918077

Also, a 3d version of the same thing has been added to the repo. by @dangarfield . You can check a live demo here:
https://hunar4321.github.io/particle-life/particle_life_3d.html#91651088029

Best

**Comment by rpominov** [comment]
So cool! Great work @dangarfield !

**Reply by skal65535** [reply]
very cool 3D version!
Maybe the particle's color should be faded out depending on the distance, to accentuate the perceived depth effect.

**Comment by hunar4321** [comment]
![nice](https://user-images.githubusercontent.com/39065447/188492401-05a31ff8-3810-4c56-a94b-79835761940c.JPG)

I liked this seed: 77163940499069
